core/state_processor_rip7560.go

```diff
@@ -71,6 +43,10 @@ type ValidationPhaseResult struct {
 	SenderValidUntil    uint64
 	PmValidAfter        uint64
 	PmValidUntil        uint64
+	// tracking the calls to the EntryPoint precompile
+	PmUsed       bool
+	EpCalls      []*EntryPointCall
```

Comment:
1. PmUsed is unused
2. I still think that placing the "epCalls" inside the VPR is wrong: every place in the code, a struct is created for specific need. here you re-use a struct we keep in memory to hold transient data. 
  yes, you can clear it after each call - but it also highlight the fact that it is not needed in the VPR itself. 
As I mentioned elsewhere, the OnEnterSuper is also not needed, and thus the OnEntry method can be on EntryPointCall struct itself.

Also, we should define how to handle multiple calls to "accept". The options are either "last call applies" or "must call exactly once".
I think we should require it is called exactly once and from the top-level, so it would mimic as much as possible a custom "return" value. This way, if we ever implement it differently using EOF return value or opcode, the semantics stays the same.

Reply:
1. Removed
2. I did remove the `EpCalls` from the `ValidationPhaseResult` struct, however this did not make the code much more elegant in my opinion as I still need a struct to hold the EntryPoint callback calls' data. I created an `EntryPointCall` but it is weird IMO. Please have a look.